more/free-programming-cheatsheets.md

```diff
@@ -386,6 +387,11 @@
 * [TensorFlow v2.0 Cheat Sheet](https://web.archive.org/web/20200922212358/https://www.aicheatsheets.com/static/pdfs/tensorflow_v_2.0.pdf) - Altoros (PDF) *(:card_file_box: archived)*
 
 
+### Terraform
+
+* [Terraform Cheatsheet](https://acloudguru.com/blog/engineering/the-ultimate-terraform-cheatsheet) - A Cloud Guru, Eric Pulsifer (HTML)
```

Comment:
```suggestion
* [Terraform Cheatsheet](https://acloudguru.com/blog/engineering/the-ultimate-terraform-cheatsheet) - Eric Pulsifer (HTML)
```

Only author name is enough i feel

Reply:
Hey Sharan! Thanks for your review! I have made the change in my latest commit. Please review it and let me know if it looks good :)